openstack/octavia/values.yaml

```diff
@@ -108,6 +108,7 @@ pxc_db:
 
 status_manager:
   health_check_interval: 120
+  failover_timeout: 10
```

Comment:
I would like to see this change in the specific regions, but not in global default variables.